.travis.yml

```diff
@@ -1,22 +1,53 @@
+env:
+  global:
+    - REPO_DIR=.
+    - BUILD_COMMIT=$TRAVIS_COMMIT
+    - PLAT=x86_64
+
+install: 
+  - git clone https://github.com/matthew-brett/multibuild
```

Comment:
should this git clone a specific version so that it doesn't automatically break in the future if multibuild is changed?

Reply:
good point, will do